scratch.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
Please remove this from the commit

Reply:
Done

Reply:
https://github.com/Sefaria/AppliedAI/pull/8/commits/c0f93c6319d4d4efcdbb04bd869be9834efdacbd